openvdb_houdini/houdini/SOP_OpenVDB_Scatter.cc

```diff
@@ -164,6 +180,12 @@ newSopOperator(OP_OperatorTable* table)
             "If enabled, scatter points in the interior region of a level set."
             " Otherwise, scatter points only in the narrow band."));
 
+    // an iso-surface offset for interior scattering
+    parms.add(hutil::ParmFactory(PRM_FLT_J, "offset", "Offset")
+        .setDefault(PRMzeroDefaults)
+        .setRange(PRM_RANGE_UI, -1.0f, PRM_RANGE_UI, 1.0f)
+        .setTooltip("The offset at which to interpret the iso surface."));
+
     parms.add(hutil::ParmFactory(PRM_SEPARATOR, "sep2", ""));
 
     // Verbose output toggle
```

Comment:
I'd recommend updating the setDocumentation() call with a brief mention of VDB Points as well.

Reply:
sure